config/jobs/kubernetes-sigs/image-builder/image-builder-ci.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes-sigs/image-builder-konfigadm:
+    - name: post-image-builder-konfigadm
+      labels:
+        preset-dind-enabled: "true"
+        preset-image-builder-gcp-creds: "true"
+        preset-image-builder-aws-creds: "true"
+        preset-image-builder-azure-creds: "true"
+        preset-image-builder-vsphere-creds: "true"
+      branches:
+        - ^master$
+      always_run: false
+      run_if_changed: "^images/konfigadm/"
+      decorate: true
+      path_alias: sigs.k8s.io/image-builder
+      max_concurrency: 1
+      spec:
+        containers:
+          - image: google/cloud-sdk:264.0.0
+            resources:
+              requests:
+                cpu: "1000m"
+            command:
+              - ./images/konfigadm/build-all.sh
```

Comment:
I don't see the referenced script at `image-builder/images/konfigadm/build-all.sh` in the repo?

Reply:
Should have a PR for that in a day or 2, I wanted to get the discussion/process started on this issue as I suspect it will be more than a few days to set up the accounts and pre-reqs